src/components/transport_manager/src/websocket_server/websocket_connection.cc

```diff
@@ -51,6 +51,7 @@ WebSocketConnection<WebSocketSession<> >::WebSocketConnection(
           std::move(socket),
           std::bind(
               &WebSocketConnection::DataReceive, this, std::placeholders::_1),
+          [this](protocol_handler::RawMessagePtr frame) { OnDataSent(frame); },
```

Comment:
@mvorobio 
Can we use one approach in this case, bind or lamda ?

Reply:
@mvorobio In this particular case I would prefer bind.

Reply:
@SNiukalov  please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/7bc1f2ce4c04d80778a833d7022995d806ce5788

Reply:
@AByzhynar could you please explain why you'd prefer `bind` here?